radicle-cob/src/backend/git/change.rs

```diff
@@ -110,16 +110,11 @@ impl change::Storage for git2::Repository {
             history_type,
         };
 
-        let (revision, blob_oids) = write_manifest(self, &manifest, &contents)?;
+        let revision = write_manifest(self, &manifest, &contents)?;
         let tree = self.find_tree(revision)?;
-        let contents = NonEmpty::collect(blob_oids.into_iter().zip(contents).map(|(oid, op)| {
-            entry::EntryBlob {
-                oid: oid.into(),
-                data: op,
-            }
-        }))
-        // SAFETY: We know it's not empty because the original `contents` is `NonEmpty`.
-        .unwrap();
+        let contents = NonEmpty::collect(contents)
+            // SAFETY: We know it's not empty because the original `contents` is `NonEmpty`.
+            .unwrap();
```

Comment:
This is redundant now so you can remove it :)

Reply:
Oh!